### PR TITLE
Translations added: Portuguese (Brazil)

### LINKED ISSUE #738: [Ticket]: Translating our Documentation
### Project

Website (potterdb.com)

### Describe the ticket you want to create

To make our Potter DB accessible to a global audience, we're reaching out to contributors like you for help in translating our documentation into multiple languages.

**Get started**
- Please drop a quick note below to let us know which language you're working on. This helps us keep track of ongoing translations and prevents duplication of efforts.
- Clone the Repository and navigate to the [docs](https://github.com/danielschuster-muc/potter-db/tree/master/docs) folder within the repository.
- Translate the `.en.mdx` files within the `/pages` directory and start translating them into your desired language.

If you encounter any challenges or need clarification while translating, don't hesitate to reach out to us. We're here to assist you every step of the way.

### Acceptance Criteria

- [ ] Translate the documentation to your language
- [ ] Add you language to the [i18n section](https://github.com/danielschuster-muc/potter-db/blob/master/docs/theme.config.tsx#L32)

### Additional Details

https://docs.potterdb.com

### Screenshots

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by cnfranco**
I translate the documentation to Portuguese (Brazil).

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes
- [x] I have updated necessary documentation

### CODE REVIEW

**Comment by danielschuster-muc** [comment]
This PR doesn't seem to be finished.

**Comment by danielschuster-muc** [comment]
Can I help you with anything @cnfranco?

**Comment by cnfranco** [comment]
> This PR doesn't seem to be finished.

Yes, it wasn't finished, but I added the latest translations throughout this week. However, I am unable to correct the following check error (image attached).

potter

Could you see what could be causing this?

**Comment by danielschuster-muc** [comment]
> > This PR doesn't seem to be finished.
> 
> Yes, it wasn't finished, but I added the latest translations throughout this week. However, I am unable to correct the following check error (image attached).
> 
> potter
> 
> Could you see what could be causing this?


<html>
<body>


Failed to compile.
--
17:12:17.916 |  
17:12:17.917 | ./theme.config.tsx:99:190
17:12:17.917 | Type error: Property assignment expected.
17:12:17.917 |  
17:12:17.917 | 97 \|     </>
17:12:17.917 | 98 \|   ),
17:12:17.917 | >  99 \|   i18n: [{ locale: "en", text: "English" }, { locale: "fr", text: "Français" }, { locale: "es", text: "Español" }, { locale: "sk", text: "Slovenčina" }, { locale: "de", text: "Deutsch" }], { locale: "pt", text: "Português" }],
17:12:17.918 | \|                                                                                                                                                                                              ^
17:12:17.918 | 100 \|   logo: <span>Potter DB: Docs</span>,
17:12:17.918 | 101 \|   project: {
17:12:17.918 | 102 \|     link: "https://github.com/danielschuster-muc/potter-db",
17:12:17.959 | Error: Command "npm run build" exited with 1


</body>
</html>

Seems to be an issue with the theme.config.tsx.

File: docs/theme.config.tsx [hunk 1]
```diff
@@ -89,7 +96,7 @@ const config: DocsThemeConfig = {
       <meta name="og:title" content="Potter DB: Docs" />
     </>
   ),
-  i18n: [{ locale: "en", text: "English" }, { locale: "fr", text: "Français" }, { locale: "es", text: "Español" }, { locale: "sk", text: "Slovenčina" }, { locale: "de", text: "Deutsch" }],
+  i18n: [{ locale: "en", text: "English" }, { locale: "fr", text: "Français" }, { locale: "es", text: "Español" }, { locale: "sk", text: "Slovenčina" }, { locale: "de", text: "Deutsch" }], { locale: "pt", text: "Português" }],
```
Comment (danielschuster-muc) [inline docs/theme.config.tsx:99]:
```suggestion
  i18n: [{ locale: "en", text: "English" }, { locale: "fr", text: "Français" }, { locale: "es", text: "Español" }, { locale: "sk", text: "Slovenčina" }, { locale: "de", text: "Deutsch" }, { locale: "pt", text: "Português" }],
``` 
This should probably fix it @cnfranco

**Comment by cnfranco** [comment]
@danielschuster-muc, can you please review?

File: docs/pages/apis/rest.pt.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  - `'false'`: falso
+  - `present`: presente 
+  - `blank`: em branco
+  - `'null'`: nulo
+  - `not_null`: não é nulo
+</details>
+
+### Classificação
+
+Você pode ordenar recursos adicionando um parâmetro `sort` da query à sua solicitação. Por exemplo, para ordenar personagens pelo nome, você pode usar o atributo `name` (vá para [Recursos](#resources) para ver todos os atributos disponíveis):
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.pt.mdx:153]:
```suggestion
Você pode ordenar recursos adicionando um parâmetro `sort` da query à sua solicitação. Por exemplo, para ordenar personagens pelo nome, você pode usar o atributo `name` (vá para [Recursos](/resources) para ver todos os atributos disponíveis):
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Looks quite good so far. On mayor thing: please always add an empty line below headers / subheaders. 
Looking forward to seeing the other missing pages.

File: docs/pages/index.pt.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+# Introdução
+
+Bem-vindo (a) à documentação de **Potter DB: API**!
+Mergulhe no mundo mágico de Harry Potter com acesso a personagens, livros, feitiços e muito mais. 
+Se você é um desenvolvedor criando um aplicativo com o tema Harry Potter ou apenas um fã de curiosidades, essa documentação irá guiá-lo através dos segredos de nossa API.
+
+## O que é Potter DB: API?
+Potter DB: API é uma interface baseada em REST e GraphQL para um vasto repositório de dados do mundo mágico de Harry Potter. Cuidamos dos dados para que você possa se concentrar na construção do seu aplicativo. Com esta API, você pode:
+- Recuperar informações detalhadas sobre seus amados `personagens`, incluindo seus backgrounds, casa de afiliação e muito mais.
+- Explorar uma lista abrangente de `feitiços`e `poções`usadas no mundo mágico.
```
Comment (danielschuster-muc) [inline docs/pages/index.pt.mdx:85]:
```suggestion
- Explorar uma lista abrangente de `feitiços` e `poções` usadas no mundo mágico.
```

File: docs/pages/getting-started.pt.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+</table>
+}
+
+## Limitação de taxa
+Para manter a qualidade e previnir usos maliciosos do Potter DB: API, há um limite baseado em ip de **900 solicitações por hora** (15 solicitações / minuto). Se você atingir esse limite, receberá um `429 To Many Requests`.
+Por favor, note que esses limites de taxa estão sujeitos a mudanças, então é essencial verificar a informação de limite de taxa mais recente na documentação ou via respostas da API. 
+
+<br />
+
+_Agora que você conhece o básico, vamos mergulhar na documentação da API para explorar endpoints disponíveis e começar a elaborar suas consultas mágicas. Refira-se a seção de documentação para [REST](/apis/rest) e [GraphQL](apis/graphql) para informações detalhadas de como usar a API efetivamente. Para uma visão geral de recursos disponíveis, confira a [Resources Overview](/resources)._
```
[the remark this answers is not in the document]
Comment (cnfranco) [reply docs/pages/getting-started.pt.mdx]: Done.

File: docs/pages/resources.pt.mdx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { Card, Cards } from 'nextra/components'
+
+# Resources Overview
```
[the remark this answers is not in the document]
Comment (cnfranco) [reply docs/pages/resources.pt.mdx]: Done.

File: docs/pages/index.pt.mdx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+- Conseguir
```
[the remark this answers is not in the document]
Comment (cnfranco) [reply docs/pages/index.pt.mdx]: Done.

File: docs/pages/getting-started.pt.mdx [hunk 7]
```diff
@@ -0,0 +1,42 @@
+import { Bleed } from 'nextra-theme-docs'
+
+# REST vs. GraphQL
```
Comment (danielschuster-muc) [inline docs/pages/getting-started.pt.mdx:3]: empty line after it

File: docs/pages/getting-started.pt.mdx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+Potter DB: API oferece duas maneiras principais de acessar nossos dados mágicos: **REST** e **GraphQL**. Escolha uma que melhor atende suas necessidades e estilo de desenvolvimento.
+- **REST**: Se você preferir uma abordagem direta e baseada em recursos, REST é o caminho a seguir. 
+- **GraphQL**: Para consultas de dados mais flexíveis e precisas, GraphQL permite que você solicite exatamente o que você precisa e nada mais. Construa suas consultas para buscar dados com controle refinado.
+
+## Technical details
```
Comment (danielschuster-muc) [inline docs/pages/getting-started.pt.mdx:8]: same here

File: docs/pages/getting-started.pt.mdx [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    </tr>
+    <tr>
+      <td>Versão atual</td>
+      <td>v1</td>
+    </tr>
+  </tbody>
+</table>
+}
+
+## Limitação de taxa
```
Comment (danielschuster-muc) [inline docs/pages/getting-started.pt.mdx:36]: and here

File: docs/pages/index.pt.mdx [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+  );
+};
+
+# Introdução
+
+Bem-vindo (a) à documentação de **Potter DB: API**!
+Mergulhe no mundo mágico de Harry Potter com acesso a personagens, livros, feitiços e muito mais. 
+Se você é um desenvolvedor criando um aplicativo com o tema Harry Potter ou apenas um fã de curiosidades, essa documentação irá guiá-lo através dos segredos de nossa API.
+
+## O que é Potter DB: API?
```
Comment (danielschuster-muc) [inline docs/pages/index.pt.mdx:82]: here

File: docs/pages/index.pt.mdx [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+Potter DB: API é uma interface baseada em REST e GraphQL para um vasto repositório de dados do mundo mágico de Harry Potter. Cuidamos dos dados para que você possa se concentrar na construção do seu aplicativo. Com esta API, você pode:
+- Recuperar informações detalhadas sobre seus amados `personagens`, incluindo seus backgrounds, casa de afiliação e muito mais.
+- Explorar uma lista abrangente de `feitiços` e `poções` usadas no mundo mágico.
+- Conseguir detalhes completos sobre os `livros` e `filmes` do universo Harry Potter. 
+
+## Estatística
```
Comment (danielschuster-muc) [inline docs/pages/index.pt.mdx:88]: here

File: docs/pages/apis/graphql.pt.mdx [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  }
+  character(slug: "harry-potter") {
+    name
+  }
+}
+```
+Nessa query, estamos solicitando o `title` e o `releaseDate` dos primeiros três livros, assim como o `title` e o `slug` de todos os capítulos de cada livro. Adicionalmente, estamos solicitando o `name`do personagem com o identificador `harry-potter`.
+
+## Esboço
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.pt.mdx:46]: empty line after heading pls

File: docs/pages/apis/graphql.pt.mdx [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+<Callout type="warning" emoji="⚠️">
+  A API GraphQL ainda está em andamento e algumas funcionalidades como filtrar e ordenar ainda **não** estão implementadas no momento! São esperadas mudanças na API.
+</Callout>
+
+# API GraphQL
+
+Nossa API GraphQL fornece uma maneira flexível e poderosa de interagir com o Potter DB. Esta seção introduzirá você a aspectos-chave de nossa API GraphQL e como aproveitar suas capacidades.
+
+## GraphQL Endpoint
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.pt.mdx:11]: Could you please always add empty lines after the headings

File: docs/pages/apis/graphql.pt.mdx [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+Para fazer consultas e mudanças, envie solicitações `POST` para o endpoint GraphQL (`/graphql`) com suas consultas e variáveis no corpo da solicitação. A API espera que o corpo da solicitação esteja no formato JSON.
+
+## Consultando Dados
+GraphQL permite solicitar precisamente os dados que você precisa. Para construir uma query, especifique os campos que você quer recuperar e seus relacionamentos. Por exemplo:
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.pt.mdx:15]: also here

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Please always add an empty line after the Headers. 
E.g.:

```
# Title

and here comes the text after one empty line
```

Also don't forget to translate the resource pages for books, chapter, characters etc.

**Comment by cnfranco** [comment]
> Please always add an empty line after the Headers. E.g.:
> 
> ```
> # Title
> 
> and here comes the text after one empty line
> ```
> 
> Also don't forget to translate the resource pages for books, chapter, characters etc.

Done.

**Comment by danielschuster-muc** [comment]
> Also don't forget to translate the resource pages for books, chapter, characters etc.

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Resources sub pages are still missing

**Pull request closed without merging**